### PR TITLE
Fixes overflow in sticky diff header when shrinking page

### LINKED ISSUE #146: Check for overflow in sticky diff header on page resize
[Reference](https://github.com/sindresorhus/refined-github/pull/128#issuecomment-204201479)

### PR DESCRIPTION
**Pull request by DrewML**
Closes #146

### CODE REVIEW

File: extension/utils.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+			};
+			const callNow = immediate && !timeout;
+			clearTimeout(timeout);
+			timeout = setTimeout(later, wait);
+			if (callNow) {
+				func.apply(this, args);
+			}
+		};
+	}
+};
```
Comment (sindresorhus) [inline extension/utils.js]: I think we should use the following pattern for exporting stuff:

```js
(exports => {
  'use strict';

  exports.debounce = () => { ... }
})(window.utils = {});
```

The makes it more Node.js like.

File: extension/manifest.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
 			"js": [
 				"vendor/sprint.min.js",
 				"vendor/gh-injection.js",
+				"utils.js",
```
Comment (sindresorhus) [inline extension/manifest.json]: Call it `util.js`

**Comment by DrewML** [comment]
@sindresorhus Updates pushed

**Comment by sindresorhus** [comment]
Looks good. Nice work @DrewML :)

**Pull request merged**